docs/architecture/product_modelling/backfilling.md

```diff
@@ -0,0 +1,77 @@
+# Backfilling Existing Subscriptions
+When updating a product block that already exists in your orchestrator, it could be the case that new attributes are
+added or removed. When removing resource types from a `ProductBlock`, the database migration generator is able to pick
```

Comment:
> the database migration generator

This refers to the `migrate-domain-models` command, right?

Maybe it's better to write it as such so people can find it (I can't find it when I search "Database migration generator")